.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Related Issue
+<!--- This project only accepts pull requests related to open issues -->
```

Comment:
Really?  Will you _ONLY_ accept pull requests related to open issues?  Seems draconian.  Some stuff just doesn't need an Issue.  Maybe say instead that this project is more likely to accept pull requests related to open issues?

Reply:
We've previously had a policy that code contributions need to be associated with a redmine ticket. That rule existed so that users could just watch redmine and know what was being changed. If we ensure there is an issue open for any PRs, then users can just watch the issues. The discussions on the PRs can get very technical and not what most users are interested in.

Reply:
This requirement should be discussed by a bigger group of people. @jlaura maybe a topic for open source WG.